torch/csrc/jit/mobile/compatibility/model_compatibility.cpp

```diff
@@ -411,5 +413,27 @@ ModelCompatCheckResult is_compatible(
   return result;
 }
 
+void appendUpgraderFunctions(mobile::Function* function) {
+  for (auto& byteCodeFunctionWithOperator : getUpgraderBytecodeList()) {
+    function->append_function(byteCodeFunctionWithOperator.function);
+  }
+}
+
+uint64_t get_min_operator_version_via_bytecode(const mobile::Module& module) {
```

Comment:
Can we rename this to something else? This version is calculated based on operator version map, not specifically for bytecodes.